### PR TITLE
BUG fix lazy inverse computation

### LINKED ISSUE #68: find=False option on sky2image broken
It appears the find=False option is broken in the latest esutil.

### PR DESCRIPTION
**Pull request by beckermr**
This PR fixes a bug where upon read the lazy inverse was being marked as computed even if it was not. 

closes #68

### CODE REVIEW

**Comment by beckermr** [comment]
cc @esheldon for review!

File: esutil/wcsutil.py [hunk 1]
```diff
@@ -1043,10 +1043,9 @@ def ExtractDistortionModel(self):
                 self.distort["bp"] = bp
                 self.distort["bp_order"] = bporder
 
-                # If inverse can't be computed, treat it as if it
-                # already were computed
-                if cap == 0 and cbp == 0:
-                    self._inverse_computed = True
+                # If inverse can't be found, make sure we computed it later
```
Comment (esheldon) [inline esutil/wcsutil.py:1046]: probably should be "make sure we compute it later"

**Review by esheldon — CHANGES_REQUESTED** [review]
looks good to me, but I'm curious what @erykoff thinks

File: esutil/wcsutil.py (hunk 1 above)
Comment (beckermr) [inline esutil/wcsutil.py:1046]:
```suggestion
                # If inverse can't be found, make sure we compute it later
```

**Comment by erykoff** [comment]
I'm trying to follow what triggers this and what's the manifestation of the bug...

**Comment by beckermr** [comment]
Just plug any DECam finalcut image into the WCS class and try to get the inverse with find=False.

**Comment by beckermr** [comment]
The manifestation is that the output coordinates in x,y are manifestly incorrect. It always returns crpix!

**Comment by erykoff** [comment]
And with this change it will `Raise` instead?

**Comment by beckermr** [comment]
Nope. It correctly computes the inverse lazily and returns the right coordinates.

**Comment by erykoff** [comment]
Then I clearly completely misunderstood the original code.  So this is fine.

**Review by esheldon — APPROVED** [review]

**Pull request merged**